packages/admin/cms-admin/src/redirects/RedirectsTable.tsx

```diff
@@ -97,25 +95,114 @@ export function RedirectsTable({ linkBlock, scope }: Props): JSX.Element {
+                            content={linkBlock.previewContent(state)}
+                        />
+                    </TargetWrapper>
+                );
+            },
+            sortable: false,
+            flex: 1,
+        },
+        {
+            field: "comment",
+            headerName: intl.formatMessage({ id: "comet.pages.redirects.redirect.comment", defaultMessage: "Comment" }),
+            renderCell: (params) => <div>{params.value}</div>,
+            sortable: false,
+            flex: 1,
+        },
+        {
+            field: "generationType",
+            headerName: intl.formatMessage({
+                id: "comet.pages.redirects.redirect.generationType",
+                defaultMessage: "GenerationType",
+            }),
+            renderCell: (params) => (
+                <Typography>
+                    {params.value === "manual" ? (
+                        <FormattedMessage id="comet.redirects.redirect.generationType.manual" defaultMessage="Manual" />
+                    ) : (
+                        <FormattedMessage id="comet.redirects.redirect.generationType.automatic" defaultMessage="Automatic" />
+                    )}
+                </Typography>
+            ),
+            sortable: false,
+            flex: 1,
+        },
+        {
+            field: "activeness",
+            headerName: intl.formatMessage({
+                id: "comet.pages.redirects.redirect.activation",
+                defaultMessage: "Activation",
+            }),
+            renderCell: (params) => <RedirectActiveness redirect={params.row} />,
+            sortable: false,
+            flex: 1,
+        },
+        {
+            field: "actions",
+            headerName: "",
+            renderCell: (params) => (
+                <IconWrapper>
+                    <IconButton
+                        onClick={() => {
+                            stackApi.activatePage("edit", params.id.toString());
+                        }}
+                    >
+                        <Edit color="primary" />
+                    </IconButton>
+                    <TableDeleteButton
+                        icon={<DeleteIcon />}
+                        mutation={deleteRedirectMutation}
+                        refetchQueries={[namedOperations.Query.PaginatedRedirects]}
+                        selectedId={params.id.toString()}
+                        text=""
+                    />
+                </IconWrapper>
+            ),
+            sortable: false,
+            flex: 1,
+            disableColumnMenu: true,
+        },
+    ];
+
+    const dataGridProps = { ...useDataGridRemote(), ...usePersistentColumnState("RedirectsGrid") };
+    const sortModel = dataGridProps.sortModel;
+
+    const { data, loading, error } = useQuery<GQLPaginatedRedirectsQuery, GQLPaginatedRedirectsQueryVariables>(paginatedRedirectsQuery, {
         variables: {
+            ...muiGridFilterToGql(columns, dataGridProps.filterModel),
```

Comment:
the api you created isn't compatible with the default filter api as it is generated by the new crud-generator and supported by muiGridFilterToGql. That is fine, but you can't use muiGridFilterToGql then.

Reply:
have removed it

Reply:
In my opinion we should make the new API in the style of the default api generated by the crud-generator. The redirects api is used by our developers in applications, and therefore it makes sense to provide an api that is similar to all other apis in the applications. Furthermore, I believe doing so would make the implementation easier

Reply:
The api for the redirects now matches the style of the api generated by the crud-generator.